experiments/ClimaEarth/components/ocean/oceananigans.jl

```diff
@@ -53,7 +53,7 @@ function OceananigansSimulation(area_fraction, start_date, stop_date; output_dir
     resolution_points = (360, 160, 32)
     Nz = last(resolution_points)
     depth = 4000 # meters
-    z = CO.ExponentialCoordinate(Nz, -depth; scale = 0.85 * depth)
+    z = CO.ExponentialCoordinate(Nz, -depth, 0; scale = 0.85 * depth)
```

Comment:
sorry I missed this; so with ClimaOcean v0.8 `ExponentialCoordinate` moved _out_ of ClimaOcean and it's in Oceananigans now. So you need `OC.ExponentialCoordinate(...)` and also Oceananigans v0.97.2 or later.

Reply:
Thank you! I just opened a PR to fix it. Should it be `OC.Grids.ExponentialCoordinate` or `OC.ExponentialCoordinate`? It looks like ClimaOcean uses both